### DISCUSSION TITLE
游戏设置界面操作

### DISCUSSION THREAD

**Discussion by 0kk470** [jynew/jynew#1295] [Ideas]
**功能描述**

用于玩家修改一些自定义游戏设置，后续可能需要增加一个控制映射的页签。

**UI导航路径**

![image](https://user-images.githubusercontent.com/25216715/199186809-5a9ccba4-d53a-4b17-8577-c2b17b4f085f.png)
![image](https://user-images.githubusercontent.com/25216715/199187068-b0077561-69e8-4fb6-97c3-736f479a503b.png)

* 页签使用手柄的```LB、RB```或者键盘的```Q、E```按键切换。
* 子页面下则为正常的```上下左右```的UI导航。
* ```ESC```或者手柄的```取消```按键关闭此UI。

**Comment by 0kk470** [comment]
Done.

**Reply by citydream163** [reply]
游戏设置→通用→请选择游戏难度。这里的设定只能在进入游戏前生效，在游戏中打开这个界面只是查看，而非设定。所以这个文字要改一下。在游戏中打开的时候是《当前游戏难度》，且不可更改，而非《请选择游戏难度》。
